bitnami/postgresql/templates/statefulset.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         heritage: {{ .Release.Service | quote }}
         role: master
       {{- with .Values.master.podLabels }}
-      {{- toYaml . | indent 8 }}
+{{ toYaml . | indent 8 }}
```

Comment:
Could you please try using `nindent` instead?
```diff
       {{- with .Values.master.podLabels }}
-      {{- toYaml . | indent 8 }}
+      {{- toYaml . | nindent 8 }}
       {{- end }}
```

Reply:
Works as well as fits indeed better with the rest of the file.